doc/api/util.md

```diff
@@ -389,6 +389,9 @@ stream.write('With ES6');
 <!-- YAML
 added: v0.3.0
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: ArrayBuffers now also show the underlying Uint8Array.
```

Comment:
```suggestion
    description: ArrayBuffers now also show their contents formatted as an Uint8Array.
```

How about this? I’d prefer to avoid a term like “underlying Uint8Array” because that sounds as if there’s an actual `Uint8Array` associated with each `ArrayBuffer` instance